variants/rak2560/platformio.ini

```diff
@@ -0,0 +1,24 @@
+; The very slick RAK wireless RAK 4631 / 4630 board - Unified firmware for 5005/19003, with or without OLED RAK 1921
+[env:rak2560]
+extends = nrf52840_base
+board = wiscore_rak4631
+board_check = true
+build_flags = ${nrf52840_base.build_flags} -Ivariants/rak2560 -D RAK_4631
+  -L "${platformio.libdeps_dir}/${this.__env__}/bsec2/src/cortex-m4/fpv4-sp-d16-hard"
+  -DGPS_POWER_TOGGLE ; comment this line to disable triple press function on the user button to turn off gps entirely.
+  -DHAS_RAKPROT=1 ; Define if RAk OneWireSerial is used (disables GPS)
+build_src_filter = ${nrf52_base.build_src_filter} +<../variants/rak2560> +<mesh/eth/> +<mesh/api/> +<mqtt/>
+lib_deps = 
+  ${nrf52840_base.lib_deps}
+  ${networking_base.lib_deps}
+  melopero/Melopero RV3028@^1.1.0
+  https://github.com/RAKWireless/RAK13800-W5100S.git#1.0.2
+  rakwireless/RAKwireless NCP5623 RGB LED library@^1.0.2
+  beegee-tokyo/RAKwireless RAK12034@^1.0.0
+  https://github.com/beegee-tokyo/RAK-OneWireSerial.git
+debug_tool = jlink
+; If not set we will default to uploading over serial (first it forces bootloader entry by talking 1200bps to cdcacm)
+;upload_protocol = jlink
+extra_scripts = 
+	${env.extra_scripts}
+	../firmware/variants/rak2560/create_uf2.py	
```

Comment:
you can drop the `../firmware` part, this doesn't work if the local folder is named differently. `variants/rak2560/create_uf2.py` should pick the script up just fine. The platfirmio.ini always assumes workspace root.